test/JDBC/expected/linked_servers-vu-verify.out

```diff
@@ -221,3 +221,141 @@ GO
+IF EXISTS(SELECT * FROM sys.servers WHERE name = 'test_server')
+    EXEC sp_dropserver 'test_server', 'droplogins'
+GO
+
+EXEC sp_testlinkedserver 'test_server'
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: server "test_server" does not exist)~~
+
+
+-- Testing the connection to a existing linked server using the server name with leading spaces (should throw error)
+EXEC sp_addlinkedserver  @server = N'test_server', @srvproduct=N'', @provider=N'SQLNCLI', @datasrc=N'localhost', @catalog=N'master'
+GO
+
+EXEC sp_testlinkedserver ' test_server'
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: server " test_server" does not exist)~~
+
+
+-- Testing the connection to a existing linked server using the server name with mixed spaces (should throw error)
+EXEC sp_testlinkedserver ' test_server '
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: server " test_server" does not exist)~~
+
+
+-- Tesing the connection to a linked server for which user mapping does not exist (should throw error)
+EXEC sp_testlinkedserver 'test_server'
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: user mapping not found for "master_dbo")~~
+
+
+EXEC sp_dropserver @server = 'test_server', @droplogins = 'droplogins'
+GO
+
+-- Testing the connection to a linked server whose data source is incorrect (should throw error)
+EXEC sp_addlinkedserver  @server = N'test_server', @srvproduct=N'', @provider=N'SQLNCLI', @datasrc=N'localhos', @catalog=N'master'
+GO
+
+EXEC sp_addlinkedsrvlogin @rmtsrvname = 'test_server', @useself = 'FALSE', @rmtuser = 'jdbc_user', @rmtpassword = '12345678'
+GO
+
+EXEC sp_testlinkedserver 'test_server'
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: TDS client library error: DB #: 20012, DB Msg: server name not found in configuration files, OS #: 0, OS Msg: Success, Level: 2)~~
+
+
+EXEC sp_dropserver @server = 'test_server', @droplogins = 'droplogins'
+GO
+
+-- Testing the connection to a linked server whose catalog name is incorrect (should throw error)
+EXEC sp_addlinkedserver  @server = N'test_server', @srvproduct=N'', @provider=N'SQLNCLI', @datasrc=N'localhost', @catalog=N'maste'
+GO
+
+EXEC sp_addlinkedsrvlogin @rmtsrvname = 'test_server', @useself = 'FALSE', @rmtuser = 'jdbc_user', @rmtpassword = '12345678'
+GO
+
+EXEC sp_testlinkedserver 'test_server'
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: TDS client library error: Msg #: 911, Msg state: 1, Msg: database "maste" does not exist, Server: BABELFISH, Process: , Line: 1, Level: 16)~~
+
+
+EXEC sp_dropserver @server = 'test_server', @droplogins = 'droplogins'
+GO
+
+-- Testing the connection to a linked server with incorrect username/password (should throw error)
+EXEC sp_addlinkedserver  @server = N'test_server', @srvproduct=N'', @provider=N'SQLNCLI', @datasrc=N'localhost', @catalog=N'master'
+GO
+
+EXEC sp_addlinkedsrvlogin @rmtsrvname = 'test_server', @useself = 'FALSE', @rmtuser = 'jdbc_use', @rmtpassword = '1234567'
```

Comment:
add another test where datasrc, catalog, username are correct put password is incorrect